libs/api/domains/regulations/src/lib/dto/getRegulationsSearch.input.ts

```diff
@@ -0,0 +1,21 @@
+import { Field, InputType } from '@nestjs/graphql'
+import { IsOptional } from 'class-validator'
+
+@InputType()
+export class GetRegulationsSearchInput {
+  @Field({ nullable: true })
+  @IsOptional()
+  q!: string
```

Comment:
Should this maybe be `q?: string` since this is validated as optional?

Reply:
fixed